README.rst

```diff
@@ -6,7 +6,7 @@ A simple program which checks Python source files for errors.
 
 Pyflakes analyzes programs and detects various errors.  It works by
 parsing the source file, not importing it, so it is safe to use on
-modules with side effects.  It's also much faster.
+modules without any side effects.  It's also much faster.
```

Comment:
What side-effects have you observed that warrant this change?

Reply:
no side effects, that's why should be changed to "without any".

Reply:
If English is your native language I hope your English teachers are performing a Picard double face-palm. If your native language is not English you should probably avoid making changes to the grammar of English text.